testing/utp/docker/circleci/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:22.04
+FROM ubuntu:24.04
```

Comment:
why can't we use latest here?

Reply:
What are your thoughts? @carver seems to think that it's less stable to pin to `latest` than to a specific version

Reply:
Isn't ubuntu latest always a stable release of ubuntu? Isn't latest just the latest stable release

Reply:
![image](https://github.com/user-attachments/assets/c17dd0fc-c9d7-4794-bf26-27fcc308dc12)

Reply:
> why can't we use latest here?

That was my suggestion. The primary benefit I see of having a stable target (stable in the sense of best-tested, but also not changing), is that it eliminates uncertainty at new Ubuntu releases. By using latest, our release flow could break without warning or any change on our end, if some feature or tool works differently on a new Ubuntu release.

It's sort of like the equivalent of using a Cargo.lock file, so that we can choose when we're ready to try a new dependency. The base OS is our dependency in the release flow.

Reply:
So latest tracks the latest LTS release so currently that is 24.04 the next one will be 26.04